packages/cds-odata-annotation-converter/test/printer/csdl-to-cds.test.ts

```diff
@@ -0,0 +1,2399 @@
+                            ]
+                        );
+                    " */
+        });
+
+        test('multiple terms in single target', () => {
+            // arrange
+            const target: Target = {
+                type: 'target',
+                name: 'AdminService.Books',
+                terms: [
+                    {
+                        type: 'element',
+                        name: 'Annotation',
+                        attributes: {
+                            Term: {
+                                type: 'attribute',
+                                name: 'Term',
+                                value: 'UI.Facets'
+                            }
+                        },
+                        content: [
+                            {
+                                type: 'element',
+                                name: 'Collection',
+                                content: [],
+                                attributes: {
+                                    Type: { type: 'attribute', name: 'Type', value: 'UI.ReferenceFacet' }
+                                }
+                            }
+                        ]
+                    },
+                    {
+                        type: 'element',
+                        name: 'Annotation',
+                        attributes: {
+                            Term: {
+                                type: 'attribute',
+                                name: 'Term',
+                                value: 'UI.Facets'
+                            },
+                            Qualifier: {
+                                type: 'attribute',
+                                name: 'Qualifier',
+                                value: 'MyQualifierName'
+                            }
+                        },
+                        content: [
+                            {
+                                type: 'element',
+                                name: 'Collection',
+                                content: [],
+                                attributes: {
+                                    Type: { type: 'attribute', name: 'Type', value: 'UI.ReferenceFacet' }
+                                }
+                            }
+                        ]
+                    }
+                ]
+            };
+
+            // act
+            const result = printTarget(target);
+
+            // assert
+            expect(result).toMatchSnapshot();
+            /* "annotate AdminService.Books with @(
+                    UI.Facets : [
+                    ],
+                    UI.Facets #MyQualifierName : [
+                    ]
+                );
+            " */
+        });
+
+        test('Annotating bound Action/Function', () => {
+            // arrange
+            const target: Target = {
+                type: 'target',
+                name: 'AdminService.addRating(AdminService.Books)',
+                range: {
```

Comment:
Ranges are optional and not used. We should remove them to make the test a bit more compact.